js/libs/core.js

```diff
@@ -440,6 +446,30 @@ Fliplet.FormBuilder = (function() {
         return '';
       };
 
+      component.computed._fieldSliderMaxMinError = function() {
+        if (!this.min && !this.max) {
+          return;
+        }
+
+        if (this._componentName === 'flSlider') {
+          if (this.min >= this.max) {
+            return 'The minimum value cannot be higher than the maximum value';
+          }
+        }
+      };
+
+      component.computed._fieldSliderStepError = function() {
+        if (!this.min && !this.max && !this.step) {
```

Comment:
Use early exit. @yogeshmsharma-xoriant Does your team have difficult understanding what the early exit coding practice means?

```suggestion
        if (this._componentName !== 'flSlider' || !this.min && !this.max && !this.step) {
```

Reply:
added early exit.